### PR TITLE
enhance(admin): do not send invite emails to new users

### PR DESCRIPTION
**Pull request by danielgavrilov**
Apparently we are still asking new users to click an invite link in their emails, and then fill in a registration form.

Instead, this PR creates a user immediately which then lets them log in through their G Suite accounts, without any email invitations

We still don't want to allow arbitrary users with @ourworldindata.org G Suite accounts to log into the admin, they need to be explicitly added.

### CODE REVIEW

File: adminSiteServer/mail.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ const transporter = nodemailer.createTransport({
     },
 })
 
+// get rid of this?
 export async function sendMail(
     options: nodemailer.SendMailOptions
 ): Promise<any> {
```
Comment (danielgavrilov) [inline adminSiteServer/mail.ts:24]: It was only used for sending the invite. We can potentially remove the `nodemailer` dependency and remove the `EMAIL` global variables. (Although I'm not sure if Wordpress depends on them.)

**Review by marcelgerber — APPROVED** [review]
Looking good!

File: adminSiteServer/mail.ts (hunk 1 above)
Comment (marcelgerber) [reply adminSiteServer/mail.ts:24]: Do it!

File: adminSiteClient/UsersIndexPage.tsx [hunk 2]
```diff
@@ -51,32 +46,43 @@ class InviteModal extends React.Component<{ onClose: () => void }> {
...
+                                type="string"
+                                className="form-control"
+                                onChange={(e) =>
+                                    (this.fullName = e.currentTarget.value)
+                                }
+                                required
+                            />
+                        </div>
+                        <div className="form-group">
+                            <label>Email</label>
```
[the remark this answers is not in the document]
Comment (danielgavrilov) [reply adminSiteClient/UsersIndexPage.tsx:64]: Good idea! We can just use `type=email`.

**Pull request merged**